entities/auctioneer.py

```diff
@@ -12,6 +12,9 @@ def __init__(self, env, region):
 
         self.entities = {}
 
+        self.auctionable_shipments = []
```

Comment:
I don't think these are necessary - you can use `self.entities`.

Reply:
I think using these lists for the bids en keeping entities for the containers and shipments keeps things more organised and easier to use.